lib/diagnostic.rb

```diff
@@ -108,10 +108,17 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'Andy = Person.new(':favorite_number', ':first_name')'
+
+class Person
+  def initiatialize (num, name)
+    @favorite_number = num
+    @first_name = name
+  end
+end
```

Comment:
We were looking for the shorthand here:

`'person = {first_name: "Andy", favorite_number: 8}'`